neqo-transport/src/pmtud.rs

```diff
@@ -132,9 +132,20 @@ impl Pmtud {
         );
     }
 
+    /// Provides a [`Fn`] that returns true if the packet is a PMTUD probe.
+    ///
+    /// Allows filtering packets without holding a reference to [`Pmtud`]. When
+    /// in doubt, use [`Pmtud::is_pmtud_probe`].
+    pub fn is_probe_filter(&self) -> impl Fn(&SentPacket) -> bool {
+        let probe_state = Probe::Sent;
```

Comment:
@mxinden This would need to be `let probe_state = self.probe_state;`, and then I again get borrow checker issues.

Reply:
@mxinden I fixed this in 8d1d1cd3, let me know if that looks OK.

Reply:
My bad. Yes. Thanks!